### ISSUE TITLE
Please track issues in separate repos

### ISSUE THREAD

**Issue by johnjeffers** [aws-controllers-k8s/community#2206]
**Is your feature request related to a problem?**
It's extremely difficult to locate issues for specific controllers. It would be much simpler if you allowed people to open issues in the repo for their controller. I understand that the issues are supposed to be tagged, but many of them are not. Github's issue search is not very good, which makes the problem worse.

**Describe the solution you'd like**
Manage issues in separate repos, one per controller.

**Comment by michaelhtm** [comment]
Hi @johnjeffers, We do try our best to ensure issues are tagged with the appropriate label. Having a centralized repository for all issues allows us to do a better job at tracking them, as it would be hectic to monitor issues across 50 repos, which I presume was the reason for centralizing these issues.  
cc: @a-hilaly @rushmash91 @knottnt

**Comment by rushmash91** [comment]
Yes, AFAIK its intended to keep track of the issues and notice patterns within the bugs reported.
